notifications.json

```diff
@@ -0,0 +1,861 @@
+                        "$ref": "#/responses/trait:apiErrors:403"
+                    },
+                    "404": {
+                        "$ref": "#/responses/trait:apiErrors:404"
+                    },
+                    "429": {
+                        "$ref": "#/responses/trait:apiErrors:429"
+                    }
+                }
+            },
+            "post": {
+                "operationId": "sendMessage",
+                "summary": "Send Message",
+                "tags": [
+                    "Messages"
+                ],
+                "parameters": [
+                    {
+                        "name": "body",
+                        "in": "body",
+                        "schema": {
+                            "$ref": "#/definitions/message-input"
+                        }
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "",
+                        "schema": {
+                            "allOf": [
+                                {
+                                    "properties": {
+                                        "id": {
+                                            "type": "string",
+                                            "format": "uuid"
+                                        },
+                                        "status": {
+                                            "$ref": "#/definitions/message-status"
+                                        }
+                                    },
+                                    "required": [
+                                        "id",
+                                        "status"
+                                    ]
+                                }
+                            ],
+                            "type": "object"
+                        }
+                    },
+                    "400": {
+                        "$ref": "#/responses/trait:apiErrors:400"
+                    },
+                    "401": {
+                        "$ref": "#/responses/trait:apiErrors:401"
+                    },
+                    "403": {
+                        "$ref": "#/responses/trait:apiErrors:403"
+                    },
+                    "404": {
+                        "$ref": "#/responses/trait:apiErrors:404"
+                    },
+                    "429": {
+                        "$ref": "#/responses/trait:apiErrors:429"
+                    }
+                }
+            }
+        },
+        "/Users/{id}/messages": {
+            "parameters": [
+                {
+                    "name": "id",
+                    "in": "path",
+                    "required": true,
+                    "type": "string"
+                }
+            ],
+            "get": {
+                "operationId": "listMessagesByUser",
+                "summary": "List Messages (by user)",
```

Comment:
be careful with ID, sounds like a numeric id, naturally prone to Insecure Direct Object References. 
I would use UUID always, this at least limit the attacker area and comes handy with the logs.